AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_tools.py

```diff
@@ -71,7 +71,7 @@ def initialize_editor_object(self):
 
 def initialize_sc_editor_objects(self):
     self.sc_editor = self.editor_main_window.findChild(QtWidgets.QDockWidget, SCRIPT_CANVAS_UI)
-    self.sc_editor_main_window = self.sc_editor.findChild(QtWidgets.QMainWindow)
```

Comment:
Why was this change made to the legacy scripting_tools file? Was this meant for the new scripting tools file?

Additionally, add a message in the file header notifying anybody looking at this file that it is deprecated in favor of the new utils file

Reply:
this was likely done by mistake. I have added a note to the file header so that we can eventually phase out and delete this file.